### ISSUE TITLE
Add Neural networks code from scratch in Java

### ISSUE THREAD

**Issue by extinctsion** [extinctsion/neuralNetworkScratch#18]
Add neural network code from scratch using Java programming language.
[Hint]: You can take help from the python code. Your code should do the same thing as neuralNetwork0.py in Java.

**Comment by tejasrok007** [comment]
Is the requirement to only create a neural network code (basic implementation)from scratch using java programming language
Should i add a pull request i have completed it.
this is the code i have used if you think i have grasped what this issue is requesting? just tell me yes i will send a pull request
![image](https://github.com/user-attachments/assets/f18ddd80-c6c8-4710-baf0-db591e1d8c52)

![image](https://github.com/user-attachments/assets/99957099-76a4-45c0-a41f-d0dd64c10a86)

**Comment by extinctsion** [comment]
Yes, @tejasrok007 things are looking good. I would recommend you to wait till oct1 to count your PR as hacktoberfest accepted. Till then you can explore my different hacktoberfest repos.

**Comment by extinctsion** [comment]
In the mean while, I can assign you this task if you want.

**Comment by tejasrok007** [comment]
Should i be creating 2 different PR for java and C++ or one pr which just includes both

**Comment by extinctsion** [comment]
both works. But if you create seperate PRs, 2 counts will be there for hacktoberfest otherwise 1 will get counted
